tests/api/SubmissionsTest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+/**
+ * @file tests/api/SubmissionsTest.php
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionsTest
+ * @ingroup tests_api
+ * @see SubmissionHandler
+ *
+ * @brief Test class for the /submissions endpoint
+ */
+
+import('lib.pkp.tests.PKPApiTestCase');
+
+class SubmissionsTest extends PKPApiTestCase {
+	/**
+	 * @covers /submissions
+	 * @expectedException GuzzleHttp\Exception\ClientException
+	 */
+	public function testGetSubmissionsWithoutToken() {
```

Comment:
It looks like this test (and some of the tests below) could move into the `pkp-lib` submodule, so they'd apply to OMP as well.